ubyssey/templates/guide/2020/article-box.html

```diff
@@ -0,0 +1,8 @@
+<div class="article-box{% if style %}--{{ style }}{% endif %}{% if type %} __type-{{ type }}{% endif %}" style="background-image: url('https://img.freepik.com/free-vector/abstract-technology-particle-background_52683-25766.jpg?size=626&ext=jpg')">
```

Comment:
Best not to link directly to an asset stored on an external site (https://img.freepik.com)

Reply:
Oh it was only for a placeholder for visualization haha